migrations/versions/e9d61dbd0cb8_.py

```diff
@@ -0,0 +1,71 @@
+"""empty message
+
+Revision ID: e9d61dbd0cb8
+Revises: a5ee63d173dd
+Create Date: 2018-02-10 14:28:46.614028
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e9d61dbd0cb8'
+down_revision = 'a5ee63d173dd'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_foreign_key(None, 'access_codes', 'events', ['event_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'access_codes', 'user', ['marketer_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'booked_ticket', 'ticket', ['ticket_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'booked_ticket', 'user', ['user_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'custom_forms', 'events', ['event_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'email_notification', 'user', ['user_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'email_notification', 'events', ['event_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'eventsusers', 'events', ['event_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'eventsusers', 'user', ['user_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'microlocation', 'events', ['event_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'orders', 'user', ['marketer_id'], ['id'], ondelete='SET NULL')
+    op.create_foreign_key(None, 'orders', 'user', ['user_id'], ['id'], ondelete='SET NULL')
+    op.create_foreign_key(None, 'orders', 'discount_codes', ['discount_code_id'], ['id'], ondelete='SET NULL')
+    op.create_foreign_key(None, 'orders', 'events', ['event_id'], ['id'], ondelete='SET NULL')
+    op.create_foreign_key(None, 'session', 'tracks', ['track_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'session', 'session_type', ['session_type_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'session', 'microlocation', ['microlocation_id'], ['id'], ondelete='CASCADE')
+    op.create_foreign_key(None, 'session', 'events', ['event_id'], ['id'], ondelete='CASCADE')
+    op.create_index(op.f('ix_speakers_sessions_version_operation_type'), 'speakers_sessions_version', ['operation_type'], unique=False)
+    op.create_index(op.f('ix_speakers_sessions_version_transaction_id'), 'speakers_sessions_version', ['transaction_id'], unique=False)
+    op.create_foreign_key(None, 'tax', 'events', ['event_id'], ['id'], ondelete='CASCADE')
```

Comment:
I think the create_foreign_key changes are not related to the PR and these keys are already there in eventyay db afaik.@SaptakS Can you please check these changes :)

Reply:
Updated